animate/adapt.py

```diff
@@ -61,14 +81,16 @@ def __init__(self, mesh, metric, name=None):
         if name is None:
             name = mesh.name
         self.name = name
+        self.comm = comm
```

Comment:
*** Marking all connected comments with these stars ***

We could pass both `name` and `comm` to `super().__init__(mesh, name=name, comm=comm)` since that is where `self.mesh` is defined?

Reply:
Fixed in dab6246.